docs/quickstarts/react.mdx

```diff
@@ -41,216 +92,79 @@ Below is an example of an .env file. If you are signed into your Clerk Dashboard
 <InjectKeys>
 
 ```sh filename=".env"
-REACT_APP_CLERK_PUBLISHABLE_KEY={{pub_key}}
+VITE_CLERK_PUBLISHABLE_KEY={{pub_key}}
 ```
 
 </InjectKeys>
 
-### Configure `<ClerkProvider />`
+### Wrap your app in `<ClerkProvider />`
 
-Clerk requires your React application to be wrapped in the [`<ClerkProvider />`](/docs/components/clerk-provider) component. The `<ClerkProvider />` component provides active session and user context to Clerk's hooks and other components.
+The [`<ClerkProvider />`](/docs/components/clerk-provider) component provides active session and user context to Clerk's hooks and other components. Import it into your app by adding `import { ClerkProvider } from '@clerk/clerk-react'` at the top of your file. You will need to import your Publishable Key and provide it to your `<ClerkProvider />` using the `publishableKey` prop. 
 
-```tsx filename="app.tsx"
-import React from "react";
-import "./App.css";
-import { ClerkProvider } from "@clerk/clerk-react";
 
-if (!process.env.REACT_APP_CLERK_PUBLISHABLE_KEY) {
+```tsx filename="src/main.tsx" {5, 7, 9-11, 15, 17}
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
+import { ClerkProvider } from '@clerk/clerk-react'
+
+const PUBLISHABLE_KEY = import.meta.env.VITE_CLERK_PUBLISHABLE_KEY
+
+if (!PUBLISHABLE_KEY) {
   throw new Error("Missing Publishable Key")
 }
-const clerkPubKey = process.env.REACT_APP_CLERK_PUBLISHABLE_KEY;
 
-function App() {
-  return (
-    <ClerkProvider publishableKey={clerkPubKey}>
-      <div>Hello from clerk</div>
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
+    <ClerkProvider publishableKey={PUBLISHABLE_KEY}>
+      <App />
     </ClerkProvider>
-  );
-}
-
-export default App;
+  </React.StrictMode>,
+)
```

Comment:
This information should be included somewhere. 
Also, I used the React starter and the port is 5173; I think that's a vite thing. But we need to make sure we have the React "starter" updated before this docs tutorial is released - it needs to match this tutorial exactly

```suggestion
)

At this point, your application should use Clerk's [Account Portal](/docs/account-portal/overview) pages that are available out of box. Start your React application via `npm run dev`, visit `http://localhost:5173`, and sign up to get access to your application. 
```

Reply:
Yeah, 5173 is a Vite thing. Good addition.